notebooks/community/vision_ai/video_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Is there a name for the dataset. What type of dataset is it (classification, etc). What does it predict.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I added more comments about how we use this dataset, basically our product is to demo that we can run some processing (speech transcript, text detection, embedding ) and build index and search.

Reply:
There isn't a name for the dataset.